spec/steps/case_details/form_spec.rb

```diff
@@ -23,6 +24,7 @@
   let(:assigned_counsel) { 'yes' }
   let(:agent_instructed) { 'yes' }
   let(:remitted_to_magistrate) { 'no' }
+  let(:remitted_to_magistrate_date) { Date.new(2023, 4, 1) }
```

Comment:
this should be `nil` given that `remitted_to_magistrate` is no as the default as that is what valid data would look like

Reply:
thx